php/class-plugin.php

```diff
@@ -291,6 +291,8 @@ public function process_post_query_vars( $post_query_vars ) {
 			'order',
 		);
 
+		$allowed_query_vars = apply_filters( 'customize_object_selector_query_posts_allowed_vars', $allowed_query_vars );
```

Comment:
Maybe we should throw an error in case this returns `null` since then all the vars would be allowed. Or maybe in case of `null` we should set it to `array()` which will throw an error with `$extra_query_vars`. Thoughts?